src/components/manage/Widgets/QuerystringWidget.jsx

```diff
@@ -263,6 +263,9 @@ class QuerystringWidget extends Component {
       intl,
     } = this.props;
 
+    // SSR fallback
+    if (Object.keys(indexes).length === 0) return '';
```

Comment:
LGTM, but can you please move it next to the main return? Then we keep all the return together.